bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/api/AwattarApi.java

```diff
@@ -141,6 +144,18 @@ public SortedSet<AwattarPrice> getData() throws AwattarApiException {
                     double netMarket = d.marketprice / 10.0;
                     double grossMarket = netMarket * vatFactor;
                     double netTotal = netMarket + basePrice;
+
+                    // in Austria the fee is added as absolute value
+                    if ("AT".equals(country)) {
+                        // add 3% absolute fee for the aWATTar service (Ausgleichskomponente)
+                        netTotal += Math.abs(netTotal) * 0.03;
+                    } else if ("DE".equals(country)) {
+                        // add 3% fee for the aWATTar service (Ausgleichskomponente)
+                        netTotal = netTotal * 1.03;
+                    }
+
+                    netTotal = netTotal + 1.5; // add 1.5 €/MWh for aWATTar service (Beschaffungskomponente)
```

Comment:
Must be €cent/kWh

Reply:
Couldn't the user just add this to the base price which is already there? Otherwise, please make it configurable also.

Reply:
This could probably made with the base price, I'll remove it.